projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrServiceV2.java

```diff
@@ -29,40 +32,74 @@
 @Path(CoordConsts.SVC_CFG_WORK_MGR2)
 @Produces(MediaType.APPLICATION_JSON)
 public class WorkMgrServiceV2 {
+
+  private BatfishLogger _logger = Main.getLogger();
+
+  @DefaultValue(CoordConsts.DEFAULT_API_KEY)
+  @HeaderParam(CoordConsts.SVC_KEY_API_KEY)
+  private String _apiKey;
+
   /** Information on the URI of a request, injected by the server framework at runtime. */
   @Context private UriInfo _uriInfo;
 
   /** Returns the list of {@link Container containers} that the given API key may access. */
   @GET
   @Path("/containers")
-  public Response getContainers(
-      @DefaultValue(DEFAULT_API_KEY) @HeaderParam(SVC_KEY_API_KEY) String apiKey) {
-    List<Container> containers = Main.getWorkMgr().getContainers(apiKey);
+  public Response getContainers() {
+    _logger.info("WMS:getContainers\n");
+    List<Container> containers = Main.getWorkMgr().getContainers(_apiKey);
     return Response.ok(containers).build();
   }
 
   /** Redirect to /containers if the user does not supply a container ID. */
   @GET
   @Path("/container")
-  public Response redirectContainer(
-      @DefaultValue(DEFAULT_API_KEY) @HeaderParam(SVC_KEY_API_KEY) String apiKey) {
+  public Response redirectContainer() {
+    _logger.info("WMS:redirect container\n");
     return Response.status(Status.MOVED_PERMANENTLY)
         .location(_uriInfo.getRequestUri().resolve("containers"))
         .build();
   }
 
-  /** Returns information about the given {@link Container}, provided this user can access it. */
+  /** Get information about the Batfish environment and supported methods. */
   @GET
-  @Path("/container/{id}")
-  public Response getContainer(
-      @DefaultValue(DEFAULT_API_KEY) @HeaderParam(SVC_KEY_API_KEY) String apiKey,
-      @PathParam("id") String id) {
-    if (!Main.getAuthorizer().isAccessibleContainer(apiKey, id, false)) {
-      // TODO: put a proper error entity here.
-      return Response.status(Status.NOT_FOUND).build();
+  @Path("/info")
+  public Response getInfo() {
```

Comment:
I don't like the name of this method. It sounds too generic and not obviously different from e.g. `getStatus`. Please make it more informative. If for instance it is supposed to return information about an `Environment`, then it could be called `getEnvironmentInfo`.